### PR TITLE
Allow customizable breakpoints for navbars

### PR DESCRIPTION
**Pull request by pine3ree**
We usually do not need so many breakpoints for navbars as for grids: see https://github.com/twbs/bootstrap/issues/36517 for feature description.

### CODE REVIEW

File: scss/_variables.scss [hunk 1]
```diff
@@ -1080,6 +1080,8 @@ $nav-pills-link-active-bg:          $component-active-bg !default;
 // Navbar
 
 // scss-docs-start navbar-variables
+$navbar-breakpoints:                $grid-breakpoints !default
```
Comment (mdo) [inline scss/_variables.scss:1083]: This change should now go in the `_maps.scss` style sheet as it’s a map that reuses an existing one.

File: scss/_variables.scss [hunk 2]
```diff
@@ -1082,7 +1082,6 @@ $nav-pills-link-active-bg:          $component-active-bg !default;
 
 // Navbar
 
-// scss-docs-start navbar-variables
```
Comment (mdo) [inline scss/_variables.scss:1085]: This line should be restored.

File: scss/_maps.scss [hunk 3]
```diff
@@ -6,6 +6,10 @@
 $theme-colors-rgb: map-loop($theme-colors, to-rgb, "$value") !default;
 // scss-docs-end theme-colors-rgb
 
+// scss-docs-start navbar-breakpoints
```
Comment (mdo) [inline scss/_maps.scss:9]: Should we document this on the navbar page as well?

**Comment by pine3ree** [comment]
Closed in favor of https://github.com/twbs/bootstrap/pull/38909

**Pull request closed without merging**